lib/pubid/iso/identifier.rb

```diff
@@ -97,17 +97,17 @@ def urn
     def formatted(format)
       case format
       when :ref_num_short
-        to_s(with_language_code: :single, stage_format_long: false)
+        to_s(with_language_code: :single, stage_format_long: false, with_prf: true)
```

Comment:
@mico can we also allow `with_prf:` in `formatted`, or better yet, allow the `formatted` options inside `to_s`?

Reply:
> @mico can we also allow `with_prf:` in `formatted`, or better yet, allow the `formatted` options inside `to_s`?

Could you please check if that is what you mean?

Reply:
Can we do:
```
pubid.to_s(format: :ref_num_short)
# and
pubid.to_s(format: :ref_num_short, with_prf: true)
```
?